mutations/mql/CreateMqlQuery.ts

```diff
@@ -14,6 +14,7 @@ const mutation = gql`
     $pctChange: PercentChange
     $startTime: String
     $timeGranularity: TimeGranularity
+    $trimIncompletePeriods: Boolean
```

Comment:
This file is the only one I changed manually; the rest is a result of running `yarn schema && yarn codegen`.

Reply:
You'll need to add `trimIncompletePeriods` to the `useMqlQuery` hook args as well.

Reply:
DOH! Thanks!